src/sentry/static/sentry/app/components/events/groupingInfo.jsx

```diff
@@ -248,7 +248,6 @@ class EventGroupingInfo extends AsyncComponent {
   static propTypes = {
     api: PropTypes.object,
     organization: SentryTypes.Organization.isRequired,
-    group: SentryTypes.Group.isRequired,
```

Comment:
@markstory this was a stray proptype that needed to be removed. a follow up to https://github.com/getsentry/sentry/pull/13422